packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1925,7 +1931,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     public async getRootDataStore(id: string, wait = true): Promise<IFluidRouter> {
-        const context = await this.dataStores.getDataStore(id, wait);
+        const internalId = this.maybeAlias(id);
+        const context = await this.dataStores.getDataStore(internalId, wait);
         assert(await context.isRoot(), 0x12b /* "did not get root data store" */);
```

Comment:
This should not be an assert. This should be a usage error.

Reply:
I would actually keep this as an assert. This should not happen. I mean other than keeping the id of root datastore, there's no id for regular datastores. So this method can only be called with garbage ids which would be acceptable for an assert imo